### PR TITLE
Project to learn Django

### PR DESCRIPTION
**Pull request by hothanhtuan1209**

### CODE REVIEW

File: Django/polls/tests.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import datetime
+from django.urls import reverse
+from django.test import TestCase
+from django.utils import timezone
+
+from .models import Question
+
+class QuestionModelTests(TestCase):
```
Comment (cuongdoan-agilityio) [inline Django/polls/tests.py:9]: Add docstring for class.

File: Django/polls/tests.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            response.context["latest_question_list"],
+            [question],
+        )
+
+    def test_two_past_questions(self):
+        """
+        The questions index page may display multiple questions.
+        """
+        
+        question1 = create_question(question_text="Past question 1.", days=-30)
```
Comment (cuongdoan-agilityio) [inline Django/polls/tests.py]: update the variable name

File: Django/polls/tests.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        question2 = create_question(question_text="Past question 2.", days=-5)
+        response = self.client.get(reverse("polls:index"))
+        self.assertQuerySetEqual(
+            response.context["latest_question_list"],
+            [question2, question1],
+        )
+
+class QuestionDetailViewTests(TestCase):
```
Comment (cuongdoan-agilityio) [inline Django/polls/tests.py:111]: Add docstring for all class

File: Django/polls/tests.py (hunk 1 above)
Comment (hothanhtuan1209) [reply Django/polls/tests.py:9]: Done

File: Django/polls/tests.py (hunk 2 above)
Comment (hothanhtuan1209) [reply Django/polls/tests.py]: Done

File: Django/polls/tests.py (hunk 3 above)
Comment (hothanhtuan1209) [inline Django/polls/tests.py:134]: Done

**Pull request merged**